examples/all-clusters-app/all-clusters-common/src/dishwasher-alarm-stub.cpp

```diff
@@ -29,36 +31,36 @@ class DishwasherAlarmDelegate : public Delegate
 public:
     /**
      *   @brief
-     *   This method for a server whether is able to enable a currently suppressed alarm,
-     *   or suppress a currently enabled alarm
-     *   @param[in] mask The value of modify Alarm mask
-     *   @return true
-     *   @return false
+     *   A notification that the Mask attribute has changed.  When this happens, some previously suppressed
```

Comment:
So hold on.  This is a notification that someone would _like_ to check the Mask attribute.  It has not yet changed when this callback is called, right?

Reply:
Sure, how about A notification that the Mask attribute will be checked.

Reply:
"checked" or "changed"?